### ISSUE TITLE
Feature Request - Add "Collect" and/or "Top Up" to the stream details page

### ISSUE THREAD

**Issue by bordumb** [drips-network/app#255]
## Overview

When a streamer or collector visits the Stream details page, there aren't met with any "action"
It's just a static page with metadata, so a bit of a dead-end 

I'd recommend we add the option to (a) collect or (b) top-up, depending on what role the user is

![image](https://user-images.githubusercontent.com/3743841/209233845-f833debe-597d-427d-9da7-fb2b63090644.png)

Example: https://drips-app-v2-radicle.vercel.app/app/1156865424642207062204157174161146191076401024539/tokens/0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6/streams/750398276

## Proposal

1. Add these components to that page 
2. Get rid of redundant info 

<img width="809" alt="image" src="https://user-images.githubusercontent.com/3743841/209236763-db67fac0-f04d-4e82-8da0-d837555601a2.png">

## Overarching Comment

It might be worth it if we take a holistic pass across the entire app to make sure that all the pages "talk" to each other and "speak to" the overall features of the app, which is mainly (a) streaming and (b) collecting 

Basically make sure we get rid of these sort of dead-ends.

**Comment by efstajas** [comment]
#240

**Comment by efstajas** [comment]
I'm closing this for now because I think that doing this might adversely blur the lines between balances being global across streams and specific to one particular stream, which we'd rather be clear. I think as-is, the UI structure more clearly implies the distinction. If you feel strongly otherwise, please feel free to re-open the issue anytime 👍
